task2_Gal_Malka/src/Position.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Position.h"
+
+Position::Position(const string& position){ 
+	row = int(position[0] - 'a');
+	col = int(position[1] - '1'); 
+}
+
+Position::Position(const int row, const int col) { 
+	this->row = row; 
+	this->col = col; 
+}
+
+Position::Position(const Position& other) { 
+	row = other.row;
+	col = other.col; 
+}
+
+Position::Position(Position&& other) noexcept { 
```

Comment:
Since position has only trivial members (it contains simply two ints), I don't think there is a need for a move constructor. The compiler will just make a default one for you since these types are trivially movable